index.html

```diff
@@ -2,21 +2,25 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
     <script src="https://kit.fontawesome.com/d52fa491d2.js" crossorigin="anonymous"></script>
+    <title>Document</title>
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
```

Comment:
- [ ] There is white space in the right side of the header and headline please try to remove it in order to match the Figma design 
![image](https://user-images.githubusercontent.com/61790587/177769800-7605d59c-e8be-4a75-bd65-0c8c58fb5d3b.png)